### DISCUSSION TITLE
About deployment to AWS

### DISCUSSION THREAD

**Discussion by levu304** [blitz-js/blitz#3067] [Q&A]
Hi, i'm having a Next.js app that deployed to AWS Lambda@Edge Serverless and a .NET server on Lambda. Now i wanna migrate to Blitz js app to remove .NET server.
https://github.com/serverless-nextjs/serverless-next.js

So did anyone deploy it to AWS Lambda@Edge Serverless yet? Is it similar to Next.js?

**Comment by beerose** [comment]
Hi! You can check out this: https://github.com/serverless-nextjs/serverless-next.js
